.github/workflows/node.js.yml

```diff
@@ -16,15 +16,17 @@ jobs:
 
     strategy:
```

Comment:
Since you removed the reference to matrix.node-version and setup-node/@v2 I don't believe you need thit strategy: section at all, as the action is just using the npm installed on the default Docker image used to run actions.